.dockerignore

```diff
@@ -0,0 +1,19 @@
+# Files to ignore when building Docker image
+
+# Git data directory
+.git
+
+# Committed files that are not needed in the build
+docker-compose.yml
+README.md
+LICENSE
```

Comment:
I think the `LICENSE` file should be included, I think technically it would be breaking the license to distribute a docker image without the license included